### PR TITLE
Create readme.md for feature flags

### PR DESCRIPTION
**Pull request by mjkkirschner**
### Purpose

add a readme for the feature flags CLI tool similar to our other tools.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### CODE REVIEW

**Comment by mjkkirschner** [comment]
see the parsed mermaid js diagram at:
https://github.com/DynamoDS/Dynamo/blob/mjkkirschner-patch-1/src/Tools/DynamoFeatureFlags/readme.md

**Review by sm6srw — APPROVED** [review]
LGTM!

File: src/Tools/DynamoFeatureFlags/readme.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Diagram:
+
+
+```mermaid
+sequenceDiagram
+  Dynamo->> FeatureFlagsManager: construct instance of FeatureFlagsManager()
+  FeatureFlagsManager->>FeatureFlagsCLIWrapper: Start Process with USERID and Dynamo PROCESSID, load mobile key based on build configuration.
+  FeatureFlagsCLIWrapper->>LaunchDarkly: Connect to LD service and Request all flag data.
+  LaunchDarkly->>FeatureFlagsCLIWrapper: log all flag data for user to std.out.
```
[the remark this answers is not in the document]
Comment (mjkkirschner) [reply src/Tools/DynamoFeatureFlags/readme.md:37]: The `FeatureFlagsCLIWrapper` is of course out of process, the data is written to std.out buffer, but that doesn't mean Dynamo has read from the buffer yet, or that it's available in memory in Dynamo yet. That does not happen until after Dynamo calls `CacheAllFlags()`;

[the remark this answers is not in the document]
Comment (aparajit-pratap) [reply src/Tools/DynamoFeatureFlags/readme.md:37]: Got it.

**Review by aparajit-pratap — APPROVED** [review]

**Pull request merged**